README.md

```diff
@@ -164,4 +189,4 @@ explain why.
 
 ## A2.7
 
-«replace this with your answer»
+The simplest Chomsky grammar level for this language is Type 3. This is because this problem is strictly defined and can use reguarly generate languages to solve the problem. 
```

Comment:
# 2

I don't think the explanation actually means anything :)

You know it's type 3 because you can parse it with a simple state machine.